### PR TITLE
[PE] Add joint probability distribution

### PR DESCRIPTION
**Pull request by Calhau18**

### CODE REVIEW

**Comment by Calhau18** [comment]
RTM

File: content/pe/0005-distribuicoes-conjuntas.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
+---
+title: Distribuições Conjuntas de Probabilidade
+path: /pe/distribuicoes-conjuntas
+type: content
+---
+
+// TODO: Gaspar mete aí cores nos bolds pls
```
Comment (diogotcorreia) [inline content/pe/0005-distribuicoes-conjuntas.md:7]: @randomicecube

File: content/pe/0005-distribuicoes-conjuntas.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+\sum_{(X,Y) \in \R_{X,Y}} P(X = x, Y = y) = 1
+$$
+
+$$
+P(X = x, Y = y) > 0, \forall_{(X,Y) \in \R_{X,Y}}
+$$
+
+A função de probabilidade conjunta costuma ser representada por uma tabela de duas entradas como mostrado abaixo:
+
+// TODO: INSERIR TABELA
```
Comment (diogotcorreia) [inline content/pe/0005-distribuicoes-conjuntas.md:35]: TODO ^

File: content/pe/0005-distribuicoes-conjuntas.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+$$
+P(X=x, Y=y) = P(X=x)P(Y=y) \Leftrightarrow \\
+F_{X,Y}(x,y) = F_X(x) F_Y(y) \Leftrightarrow \\
+P(X=x|Y=y) = P(X=x) \Leftrightarrow P(Y=y|X=x) = P(Y=y) \Leftrightarrow \\
+E(Y|X=x) = E(Y) \Leftrightarrow E(X|Y=y) = E(X) \Leftrightarrow \\
+V(Y|X=x) = V(Y) \Leftrightarrow V(X|Y=y) = V(X) \\
+$$
+
+// GASPAAAAAAAAAAAAAAR mete as mesmas cores que meteste em cima :fixe2:
```
Comment (diogotcorreia) [inline content/pe/0005-distribuicoes-conjuntas.md:95]: @randomicecube :kek:

**Review by diogotcorreia — CHANGES_REQUESTED** [review]
Still incomplete

**Review by diogotcorreia — APPROVED** [review]

**Pull request merged**